src/psyclone/tests/dynamo0p3_stubgen_test.py

```diff
@@ -61,28 +61,6 @@ def setup():
     Config.get().api = "lfric"
 
 
-def test_kernel_stub_invalid_iteration_space():
```

Comment:
How come you've removed this test? The stub generator can't currently generate a stub for a kernel which operates on 'dof' so I think we still need it.

Reply:
You're absolutely right, I'll put this back

Reply:
I've added it back but had to drop 'dof' from `supported_operates_on` in `LFRicKern.gen_stub`